frontend/src/components/PostCategories.js

```diff
@@ -31,11 +33,26 @@ const PostCategories = ({ categories, preview }) => {
 
   return (
     <div className={postCategoriesClassnames}>
-      {slicedCategories.map((category, index) => (
-        <div key={index} className="post__single-category">
-          {category}
-        </div>
-      ))}
+      {slicedCategories.map((category, index) => {
+        return (
+          <Tooltip
+            key={index}
+            interactive
+            arrow
+            html={
+              <a
+                href={`${category}_link`}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {category}
+              </a>
+            }
+          >
+            <div className="post__single-category">{category}</div>
+          </Tooltip>
```

Comment:
We might extract to a separate component.

Reply:
Okey